### ISSUE TITLE
Bug: poster image always displayed

### ISSUE THREAD

**Issue by InnigerM** [muxinc/elements#793]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Mux Elements/Packages does this apply to? Select all that apply

mux-player-react

### Which browsers are you using?

Chrome, Safari, Firefox

### Which operating systems are you using?

macOS

### Description

When upgrading to `@mux/mux-player-react` > v1.12.1 I am getting the issue that a custom poster image set via `poster` property is always displayed. Probably is related to https://github.com/muxinc/elements/issues/590, since this introduced the feature for having a always displayed poster image.
Tried to also directly set `audio` prop to false, didn't help though.

### Steps to reproduce

1. Create a `<MuxPlayer>` component
2. Set `poster` property to a valid link source
3. Play the video

### Current Behavior

Poster image is always displayed if set

### Expected Behavior

Poster image is only displayed if the video isn't playing. (Probably even more intuitive for me if it's only displayed if the video hasn't started yet)

### What version of the package are you using?

1.14.0

**Comment by InnigerM** [comment]
Hi @cjpillsbury this version actually does work for me as well. Package included in our app is [this](https://www.npmjs.com/package/@mux/mux-player-react) at version 1.12.1, we're not experiencing this bug. Above that version we do, though.
Could you confirm that this is the correct package we're using?

**Comment by smorimoto** [comment]
Dig deeper into the internal implementation, you will notice that there is `MuxPlayerInternal`.

https://github.com/muxinc/elements/blob/954b2c80d7df88bb4585c46a15dd1185d56dcf9a/packages/mux-player-react/src/index.tsx#L253-L261

Dig further into that implementation and you will also notice that there is a function called `toNativeProps`

https://github.com/muxinc/elements/blob/954b2c80d7df88bb4585c46a15dd1185d56dcf9a/packages/mux-player-react/src/index.tsx#L124-L126

Dig even deeper and we finally caught the root cause of the issue.

https://github.com/muxinc/elements/blob/954b2c80d7df88bb4585c46a15dd1185d56dcf9a/packages/mux-player-react/src/common/utils.ts#L38-L56

The audio property is the boolean type, but somehow it seems to be transformed to an empty string here 🤷‍♂️
In other words, you should be able to work around the issue temporarily by silencing TypeScript and passing the string "true" to the audio property.

https://github.com/muxinc/elements/blob/954b2c80d7df88bb4585c46a15dd1185d56dcf9a/packages/mux-player-react/src/common/utils.ts#L39

**Comment by cjpillsbury** [comment]
Closing this issue due to inactivity. Can reopen or create a new issue if necessary.
